CMakeLists.txt

```diff
@@ -21,6 +25,7 @@ catkin_package(
   LIBRARIES ${LIBRARY_NAME}
   INCLUDE_DIRS include
   CATKIN_DEPENDS mpi_cmake_modules
+  DEPENDS Boost
```

Comment:
Don't we need to add a package.xml info as well on this? I think there is a special xml stuff for the system dependencies.

Reply:
You are right, there should be a `<depend>boost</depend>` in the package.xml, I'll add it.